arch/arm/boot/dts/overlays/overlay_map.dts

```diff
@@ -108,4 +108,13 @@
 	upstream-aux-interrupt {
 		deprecated = "no longer necessary";
 	};
+
+	vc4-kms-v3d {
+		bcm2835;
+		bcm2711 = "vc4-kms-v3d-pi4";
+	};
+
+	vc4-kms-v3d-pi4 {
+		bcm2711;
```

Comment:
I guess this could be implied by the property above. Although I see how it can make the implementation complicated.

Reply:
Yes it is implied, but I don't really fancy having to search the whole map for string properties that match the current name every time an overlay is applied. However, I can add a test that confirms that mappings like this are consistent.

Reply:
Fair enough :)